Apps/frontend/src/views/Waves.svelte

```diff
@@ -48,10 +48,25 @@
     }
   };
 
+  // Sets the scaling of a individual wave according to the voltage intervals
+  const setScaling = (index, scale) => {
```

Comment:
Can you please keep a consistent naming -> either scaling or amplitude.

Reply:
changed it to scaling.